src/main/kotlin/org/opensearch/replication/ReplicationPlugin.kt

```diff
@@ -127,11 +127,12 @@ import org.opensearch.threadpool.FixedExecutorBuilder
 import org.opensearch.threadpool.ScalingExecutorBuilder
 import org.opensearch.threadpool.ThreadPool
 import org.opensearch.watcher.ResourceWatcherService
+import java.util.Collections
 import java.util.Optional
 import java.util.function.Supplier
 
 internal class ReplicationPlugin : Plugin(), ActionPlugin, PersistentTaskPlugin, RepositoryPlugin, EnginePlugin {
-
+    private var isPluginDisabled: Boolean = true
```

Comment:
Lets ensure this is backported to ES 7.10 branch

Reply:
Yep.